src/python/datastore/data_handler.py

```diff
@@ -375,7 +377,7 @@ def get_formatted_reproduction_help(testcase):
   # Since this value may be in a job definition, it's non-trivial for it to
   # include newlines. Instead, it will contain backslash-escaped characters
   # that must be converted here (e.g. \n).
-  help_format = help_format.decode('unicode-escape')
+  help_format = help_format.encode().decode('unicode-escape')
```

Comment:
Why both encode and decode ?

Reply:
In Python 2 it was valid to re-encode a unicode object but that's no longer the case in 3. This transformation goes from normal utf-8 to bytes to escaped encoded unicode.